@xen-orchestra/web-core/lib/components/mobile-fullscreen/MobileFullscreen.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <slot v-if="uiStore.isDesktop || disabled" />
+  <Teleport v-else-if="isOpen" to="body">
+    <div class="mobile-fullscreen">
+      <MobileHeadBar @close="emit('close')">
+        <slot name="header" />
+      </MobileHeadBar>
+      <slot />
+    </div>
+  </Teleport>
+</template>
+
+<script lang="ts" setup>
+import MobileHeadBar from '@core/components/mobile-fullscreen/MobileHeadBar.vue'
+import { useUiStore } from '@core/stores/ui.store'
+
+defineProps<{
+  disabled?: boolean
+  isOpen?: boolean
+}>()
+
+const emit = defineEmits<{
+  close: []
+}>()
+
+const uiStore = useUiStore()
+</script>
+
+<style lang="postcss" scoped>
+.mobile-fullscreen {
+  background-color: var(--background-color-secondary);
+  position: fixed;
+  inset: 0;
+  z-index: 100;
```

Comment:
To avoid some visual conflicts, for example when the sidebar is open on mobile, you can increase the value:
```suggestion
  z-index: 1002;
```